extensions/wikidata/tests/src/org/openrefine/wikidata/qa/scrutinizers/ConflictsWithScrutinizerTest.java

```diff
@@ -33,6 +45,19 @@ public void testTrigger() {
 
         ItemUpdate updateA = new ItemUpdateBuilder(idA).addStatement(statement1).addStatement(statement2).build();
 
+        PropertyIdValue propertyIdValue = Datamodel.makeWikidataPropertyIdValue("P2302");
+        EntityIdValue entityIdValue = Datamodel.makeWikidataItemIdValue("Q21502838");
+        PropertyIdValue property = Datamodel.makeWikidataPropertyIdValue("P2306");
+        Value propertyValue = Datamodel.makeWikidataPropertyIdValue("P31");
+        PropertyIdValue item = Datamodel.makeWikidataPropertyIdValue("P2305");
+        Value itemValue = Datamodel.makeWikidataItemIdValue("Q5");
```

Comment:
Perhaps it would be worth using more informative variable names here, to emphasize the role they play in the constraint definition. You should also be able to reuse the constants declared at the beginning of `ConflictsWithScrutinizer`.